playwright-gh-pages/README.md

```diff
@@ -0,0 +1,209 @@
+        with:
+          node-version: '22'
+          cache: 'npm'
+
+      - name: Install dev dependencies
+        run: npm ci
+
+      - name: Start Grafana
+        run: |
+          docker compose pull
+          DEVELOPMENT=false GRAFANA_VERSION=${{ matrix.GRAFANA_IMAGE.VERSION }} GRAFANA_IMAGE=${{ matrix.GRAFANA_IMAGE.NAME }} docker compose up -d
+
+      - name: Wait for grafana server
+        uses: grafana/plugin-actions/wait-for-grafana@main
+
+      - name: Install Playwright Browsers
+        run: npm exec playwright install chromium --with-deps
+
+      - name: Run Playwright tests
+        id: run-tests
+        run: npm run e2e
+
+      - name: Upload e2e test summary
+        uses: grafana/plugin-actions/playwright-gh-pages/upload-report-artifacts@main
+        if: ${{ always() }}
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          test-outcome: ${{ steps.run-tests.outcome }}
+
+  deploy-pages:
+    if: ${{ always() }}
+    needs: [playwright-tests]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Publish report
+        uses: grafana/plugin-actions/playwright-gh-pages/deploy-report-pages@main
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+```
+
+### Example using a per-plugin matrix
+
+The following simplified example demonstrates how Playwright report publishing can be integrated in a mono repo where the matrix is derived for each plugin in the repo.
+
+```yaml
+name: e2e tests
+
+on:
+  pull_request:
+    branches:
+      - master
+      - main
+
+permissions:
+  contents: write
+  id-token: write
+  pull-requests: write
+
+jobs:
+  e2e:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        plugin-id: ['grafana-panel-sample1', 'grafana-panel-sample2', 'grafana-panel-sample3']
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      # necessary steps to install dependencies and build the plugin
+
+      - name: Start Grafana latest
+        run: |
+          docker compose pull
+          DEVELOPMENT=false GRAFANA_VERSION=latest GRAFANA_IMAGE=grafana-enterprise docker compose up -d
+
+      - name: Wait for grafana server
+        uses: grafana/plugin-actions/wait-for-grafana@main
+
+      - name: Install Playwright Browsers
+        run: npm exec playwright install --with-deps
+
+      - name: Run Playwright tests
+        id: run-tests-latest
+        run: npm run e2e
+
+      - name: Upload e2e test summary
+        uses: grafana/plugin-actions/playwright-gh-pages/upload-report-artifacts@main
+        if: ${{ always() }}
+        with:
+          report-dir: playwright-report
+          grafana-version: latest
+          grafana-image: grafana-enterprise
+          plugin-name: ${{ matric.plugin-id }}
+          test-outcome: ${{ steps.run-tests-latest.outcome }}
+      # repeat steps but for another Grafana version if necessary
+
+  deploy-pages:
+    if: ${{ always() }}
```

Comment:
Same as above, is this intended or do we have to exclude the case when the workflow is cancelled?